roles/kubernetes/node/vars/fedora.yml

```diff
@@ -1,2 +1,2 @@
----
+ ---
```

Comment:
Typo

Reply:
Don't worry, that's a wip to debug recent CI failure in all jobs 😄